### PR TITLE
set tunnel timeout to 60s by default in ut

### LINKED ISSUE #8380: UT fails due to tunnel timeout under high load env
## Bug Report

Please answer these questions before submitting your issue. Thanks!
some examples:
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-unit-tests/detail/tiflash-ghpr-unit-tests/11885/pipeline
https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-unit-tests/detail/tiflash-ghpr-unit-tests/11874/pipeline

### PR DESCRIPTION
**Pull request by windtalker**
### What problem does this PR solve?

Issue Number: close #8380

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by xzhangxian1008 — APPROVED** [review]
LGTM

File: dbms/src/Debug/DAGProperties.h [hunk 1]
```diff
@@ -35,6 +35,6 @@ struct DAGProperties
     UInt64 local_query_id = 1;
     Int64 task_id = 1;
 
-    Int32 mpp_timeout = 10;
+    Int32 mpp_timeout = 60;
 };
 } // namespace DB
```
Comment (xzhangxian1008) [inline dbms/src/Debug/DAGProperties.h:40]: Add line at the end of the file.

Comment (windtalker) [reply dbms/src/Debug/DAGProperties.h]: Done

**Review by yibin87 — APPROVED** [review]
LGTM

**Pull request merged**